x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/action/EsqlCapabilities.java

```diff
@@ -690,6 +690,11 @@ public enum Cap {
          */
         JOIN_LOOKUP_V11(Build.current().isSnapshot()),
 
+        /**
+         * LOOKUP JOIN without MV matching (https://github.com/elastic/elasticsearch/issues/118780)
+         */
+        JOIN_LOOKUP_SKIP_MV(JOIN_LOOKUP_V11.isEnabled()),
```

Comment:
Make it `_V12`?

Reply:
The post-merge conflicts with the renaming approach make it very prone to get main broken. So I'll either rename it later before merging, or just remove the capability and update the other one in another PR

Reply:
:+1:

Reply:
That hit me a while back too. We keep bumping the same number and then everything breaks.